secure_message/resources/threads.py

```diff
@@ -21,13 +22,78 @@ def get(thread_id):
         logger.info("Getting messages from thread", thread_id=thread_id, user_uuid=g.user.user_uuid)
 
         conversation = Retriever().retrieve_thread(thread_id, g.user)
+        conversation_metadata = Retriever.retrieve_conversation_metadata(thread_id)
 
         logger.info("Successfully retrieved messages from thread", thread_id=thread_id, user_uuid=g.user.user_uuid)
         messages = []
         for message in conversation.all():
             msg = message.serialize(g.user, body_summary=False)
             messages.append(msg)
-        return jsonify({"messages": add_users_and_business_details(messages)})
+
+        if conversation_metadata and conversation_metadata.is_closed:
+            return jsonify({"messages": add_users_and_business_details(messages),
+                            "is_closed": conversation_metadata.is_closed,
+                            "closed_by": conversation_metadata.closed_by,
+                            "closed_by_uuid": conversation_metadata.closed_by_uuid,
+                            "closed_at": conversation_metadata.closed_at})
```

Comment:
This returns a date as a string e.g. `"closed_at": "Tue, 12 Jun 2018 09:48:36 GMT"`. This should be an ISO 8601 date time e.g. `2018-06-12T09:48:36Z`

Reply:
Done